tests/onnx2pytorch/operations/test_squeeze.py

```diff
@@ -20,5 +20,5 @@ def inp():
     ],
 )
 def test_squeeze(inp, dim, exp_shape):
```

Comment:
Could you add another test with an opset_version=13?

Reply:
Done!